tensorflow_addons/image/sharpness_op.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""Sharpness Op for color ops"""
+
+import tensorflow as tf
+
+from tensorflow_addons.image.compose_ops import blend
+
+
+def sharpness(image, factor):
```

Comment:
Why not do something similar to `equalize` ops? 

Have a function called `sharpness_image` that operates on a single image. Then do a `tf.map_fn` in `sharpness` after converting to 4D?

Reply:
I haven't handled different `data_formats`  here as `NCHW` is not implemented in `depthwise_conv2d` for CPU. Batching has been added.